src/bridge/tests/token_actions_test.cairo

```diff
@@ -217,3 +333,113 @@ fn consume_message_zero_recipient() {
+    TokenBridge::constructor(ref mock, L3_BRIDGE_ADDRESS(), messaging_contract_address, OWNER());
+
+    mock.send_deploy_message(usdc_address);
+}
+
+#[test]
+#[should_panic(expected: ('L3 bridge not set',))]
+fn send_deploy_message_bridge_unset() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    mock.send_deploy_message(usdc_address);
+}
+
+#[test]
+fn send_deposit_message_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    // Deploy messaging mock with 5 days cancellation delay
+    let messaging_mock_class_hash = snf::declare("messaging_mock").unwrap();
+    // Deploying with 5 days as the delay time (5 * 86400 = 432000)
+    let (messaging_contract_address, _) = messaging_mock_class_hash
+        .deploy(@array![DELAY_TIME])
+        .unwrap();
+
+    TokenBridge::constructor(ref mock, L3_BRIDGE_ADDRESS(), messaging_contract_address, OWNER());
+
+    let no_message: Span<felt252> = array![].span();
+    mock
+        .send_deposit_message(
+            usdc_address,
+            100,
+            snf::test_address(),
+            no_message,
+            constants::HANDLE_TOKEN_DEPOSIT_SELECTOR
+        );
+}
+
+#[test]
+#[should_panic(expected: ('L3 bridge not set',))]
+fn send_deposit_message_bridge_unset() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    let no_message: Span<felt252> = array![].span();
+    mock
+        .send_deposit_message(
+            usdc_address,
+            100,
+            snf::test_address(),
+            no_message,
+            constants::HANDLE_TOKEN_DEPOSIT_SELECTOR
+        );
+}
+
+
+#[test]
+fn get_status_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    assert(mock.get_status(usdc_address) == TokenStatus::Unknown, 'Incorrect status');
+
+    // Setting the token active
+    let old_settings = mock.token_settings.read(usdc_address);
+    mock
+        .token_settings
+        .write(usdc_address, TokenSettings { token_status: TokenStatus::Active, ..old_settings });
+
+    assert(mock.get_status(usdc_address) == TokenStatus::Active, 'Incorrect status');
+
+    // Setting the token
+    let old_settings = mock.token_settings.read(usdc_address);
+    mock
+        .token_settings
+        .write(usdc_address, TokenSettings { token_status: TokenStatus::Blocked, ..old_settings });
+
+    assert(mock.get_status(usdc_address) == TokenStatus::Blocked, 'Incorrect status');
+}
+
+
+#[test]
+fn is_servicing_token_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = USDC_MOCK_ADDRESS();
+
+    // Setting the token active
+    let old_settings = mock.token_settings.read(usdc_address);
+    mock
+        .token_settings
+        .write(usdc_address, TokenSettings { token_status: TokenStatus::Active, ..old_settings });
+
+    assert(mock.is_servicing_token(usdc_address) == true, 'Should be servicing');
```

Comment:
let's check before setting the status it was false

Reply:
updated